cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,72 @@
+/* eslint-disable */
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ArticlePageObject from '../support/pages/articlePageObject'
+
+const articlePage = new ArticlePageObject()
+
 describe('Article', () => {
+  let user
+  let article
+  let newArticle
+
   before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser
+    })
 
-  });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle
+    })
+
+    cy.task('generateArticle').then((generateArticle) => {
+      newArticle = generateArticle
+    })
+  })
 
   beforeEach(() => {
-    cy.task('db:clear');
-  });
+    cy.task('db:clear')
+    cy.visit('/')
+    cy.registerAndLogin(user.email, user.username, user.password)
+    articlePage.visit()
+  })
 
   it('should be created using New Article form', () => {
-
-  });
+    articlePage.typeTitle(article.title)
+    articlePage.typeDescription(article.description)
+    articlePage.typeBodyArticle(article.body)
+    articlePage.typeTag(article.tag)
```

Comment:
The `typeTag` method is used to input tags. Verify if this method supports multiple tags if required by the application.